ocs_ci/ocs/resources/cloud_manager.py

```diff
@@ -0,0 +1,296 @@
+import base64
```

Comment:
resources dir holds classes and functions that represent CRs. I am not sure it is the correct place for this file. Maybe under ocs_ci/ocs/ dir would be better to keep the logic

Reply:
ignore, not critical